e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -40,5 +36,29 @@ Scenario: Re-enrolling an agent
 @revoke-token
 Scenario: Revoking the enrollment token for an agent
   Given an agent is deployed to Fleet
+    And the agent is un-enrolled
+    And the "agent" process is "stopped" on the host
   When the enrollment token is revoked
-  Then an attempt to enroll a new agent fails
+  Then an attempt to enroll an agent with the old token fails
+
+@package-added-to-default-config
+Scenario: Adding packages to configurations
+  Given an agent is deployed to Fleet
+    And the package list service is available
+  When the "Cisco" "latest" package version is installed successfully
+    And the "Cisco" package datasource is added to the "default" configuration
+  Then the "default" configuration shows the "Cisco" datasource added
+
+@new-agent-configuration
+Scenario: Assign an Agent to a new configuration
+  Given a new configuration named "Test Fleet" is created
+  When an agent is deployed to Fleet
+    And the Agent is assigned to the configuration "Test Fleet"
+  Then a new enrollment token is created
+
+@new-configuration-new-package
+Scenario: Add a new config and a new package and assign an agent
```

Comment:
this is covered as well with agent-endpoint integration file we got merged recently.  additionally, all tests are using a new agent policy (config), specific to the scenario so we don't need ot test this as it is is written.  we are going down the path of having at least 1 implied policy config for each Integration, if and when we want to make it more configurable it will likely still need to be enumerated as such because the policy JSON to post is a lot to try to manage as a step in the scenario